test/6_branch.js

```diff
@@ -507,7 +507,7 @@ describe('Branch', function() {
 			var assert = testUtils.plan(3, done);
 			branch.logout(function(err) {
 				assert(true);
```

Comment:
remove. I suspect this will give you a red in the test.html test.

2 lines above it said `var assert = testUtils.plan(3, done);` what this means is that we only plan to do three assertions. So the assertion below is not even run. When you remove this line, the below test will run and it should fail :(

We need better testing framework. One thing I always wanted to do but never got around to is to make it yell when there are actually more assertions than planned (it yells on fewer, however).